docs/dev/jobs-config-hail.json

```diff
@@ -0,0 +1,30 @@
+{
+  "new_cluster": {
+    "spark_version": "8.2.x-scala2.12",
```

Comment:
This isn't guaranteed to work - Glow is built for Spark 3.0, and DBR 8.2 is on Spark 3.1.